t3/utils/flux.py

```diff
@@ -304,6 +313,7 @@ def run_jsr(gas: ct.Solution,
             V: Optional[float] = None,
             a_tol: float = 1e-16,
             r_tol: float = 1e-10,
+            tau_tolerance: float = 0.1,
```

Comment:
Are we sure that `0.1` isn't too risky? In the PR comment you wrote that it reached from 0.8 to 8 sec. So if next time it will jump from 0.9 to 8 sec it would be enough for the 0.1 tolerance to miss this jump in t. We need the tolerance just for the lower range, let's make it ~0.01 or 0.005?

Reply:
was this addressed?